### DISCUSSION TITLE
Tailwind v4

### DISCUSSION THREAD

**Discussion by shellscape** [unocss/unocss#4288] [General]
hey all. did a quick search of the channel on discord (which has been crickets) and github but didn't see any results - with Tailwind v4 landing, is there any plans to support their configfileless approach within the wind preset?

**Comment by zyyv** [comment]
Tailwind4 is said in its introduction to be a ground-up rewrite of the framework. Some new features bring destructive changes. We need to think about whether this is a behavior worth aligning with, as it will break some previously set rules. 🤔

**Reply by lizyChy0329** [reply]
If you don't align with Tailwind v4, Unocss should rewrite the documentation 👍

**Reply by zyyv** [reply]
I'm trying to align something of the tailwind v4, but not all. Here is a demo https://github.com/zyyv/unocss-preset-wind-v4.
My progress isn't very fast...

**Comment by shellscape** [comment]
@zyyv why did you close this?

**Reply by zyyv** [reply]
Refer to https://github.com/unocss/unocss/issues/4411
